src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.cs

```diff
@@ -4079,6 +4079,20 @@ protected override void RescaleConstantsForDpi(int deviceDpiOld, int deviceDpiNe
                         _toolStripGrip.ToolStrip_RescaleConstants(deviceDpiOld, deviceDpiNew);
                     }
 
+                    // ToolStripItems are components and have Font property. Components does not
+                    // receive DPICHANGED messages and ToolstripItems doesn't carry parent-child
+                    // relationship with owner to get scaled by parent/Container. For these reasons,
+                    // they need to be explicitly updated for the font when Dpi changes. If font was
+                    // not explicitly set, DefaultFont, that is scaled to current Dpi, will be applied.
```

Comment:
We tend to wrap between 120 and 150 char mark:
![image](https://user-images.githubusercontent.com/4403806/121287082-4515e300-c924-11eb-8ffb-d9f69bdee768.png)

@JeremyKuhne and I use [EditorGuidelines extension](https://marketplace.visualstudio.com/items?itemName=PaulHarrington.EditorGuidelines) that shows the guidelines, and the repo is configured for it.

```suggestion
                    // ToolStripItems are components and have Font property. Components do not receive WM_DPICHANGED messages, nor they have
                    // parent-child relationship with owners and, thus, do not get scaled by parent/Container. For these reasons, they need the font
                    // to be explicitly updated when Dpi changes (only if the font was set explicitly).
```